### DISCUSSION TITLE
Auto contribute mode

### DISCUSSION THREAD

**Discussion by FurblandChannel** [ThioJoe/YT-Spammer-Purge#862] [Ideas]
I have a second computer lying around, and I’m sure some of you do too. So, here’s my idea. A mode where you can hook up a dummy account and it will retrieve urls from the recommended section and scan them, reporting any potential spam it finds. You can just leave it running on a spare machine of yours and it’ll do the rest.

**Comment by Firecul** [comment]
While I haven't read them I'm sure this would violate one of the ToS of youtube or the API. Happy to be proved wrong though if someone else has more info.
